site/_apps/nextjs-hn.txt

```diff
@@ -0,0 +1,26 @@
+---
+weight: 14
+title:  'Next.js HN'
+github-title: 'chrisdwheatley/nextjs-hn'
+libraries:
+    - name: 'Next.js'
+    - name: 'Preact'
+module-bundling: 'Next.js with additional Webpack config'
+service-worker: 'Application & data caching with sw-precache'
+perfomance-patterns: 'In memory LRU cache, DNS prefetch, Preact compat alias in production'
+server-side-rendering: 'Yes'
+api: 'Node-hnapi (unofficial)'
+hosting: 'Digital Ocean & Cloudflare'
+authors:
+    - name: 'Chris'
+lighthouse: '91/100'
+interactive-em: '4.7s'
```

Comment:
EM TTI according to the Lighthouse report is 4,410ms - We can update this to 4.4s here. 

![image](https://user-images.githubusercontent.com/110953/26901354-6eb2fa48-4b8a-11e7-8a6f-9af4bf66a049.png)